src/main/java/spotify/api/impl/BrowseApiRetrofit.java

```diff
@@ -83,6 +89,34 @@ public PlaylistSimplifiedPaging getCategoryPlaylists(String categoryId, Map<Stri
         }
     }
 
+    @Override
+    public Paging<PlaylistSimplified> getCategoryPlaylistsUnwrapped(String categoryId, Map<String, String> options) {
+        options = ValidatorUtil.optionsValueCheck(options);
+
+        logger.trace("Constructing HTTP call to fetch category playlists.");
+        Call<PlaylistSimplifiedPaging> httpCall = browseService.getCategoryPlaylists("Bearer " + this.accessToken, categoryId, options);
+
+        try {
+            logger.info("Executing HTTP call to fetch category playlists.");
+            logger.debug("Fetching category {} playlists with following values: {}.", categoryId, options);
+            LoggingUtil.logHttpCall(logger, httpCall);
+            Response<PlaylistSimplifiedPaging> response = httpCall.execute();
+
+            ResponseChecker.throwIfRequestHasNotBeenFulfilledCorrectly(response, HttpStatusCode.OK);
+
+            if(response.body() != null) {
+                logger.info("Category playlists have been successfully fetched.");
+                return response.body().getPlaylists();
+            }
+
+            logger.error(EMPTY_RESPONSE_BODY_UNKNOWN_REASON);
```

Comment:
A defensive check makes more sense and is a bit cleaner.
```
if (response.body() == null) {
// handling
}

return response.body().getPlaylists();
```

Reply:
Thanks, the defensive checks have been implemented.